### ISSUE TITLE
Which speakers are seen in the pre-trained model?

### ISSUE THREAD

**Issue by enhuiz** [cyhuang-tw/AdaIN-VC#9]
Hi, thanks very much for the sharing the model. I wonder when you trained the model, did you use 89 speakers in VCTK as training set (seen speakers) and 20 as the test set (unseen speakers)? If so, may you share which 20 speakers are in the test set?

**Comment by cyhuang-tw** [comment]
Hi,

Sorry for the late reply.
I used 80% of VCTK speakers for training and 20% for validation (in ```train.py```, line 35). However, since random_split was used for splitting, I am not sure which speakers were used in training.
I will record this information in the new version, or you can make a pull request for this!

**Comment by cyhuang-tw** [comment]
Hi,

Sorry for the late reply.
I used 80% of VCTK speakers for training and 20% for validation (in ```train.py```, line 35). However, since random_split was used for splitting, I am not sure which speakers were used in training.
I will record this information in the new version, or any pull request is welcomed.

**Comment by enhuiz** [comment]
I see, thanks.
